docs-preview/src/extension.ts

```diff
@@ -13,6 +13,7 @@ import {
     ViewColumn,
     window,
     workspace,
+    WebviewPanel,
```

Comment:
Hi @qinezh , I ran into this error while running `npm run vscode:prepublish`

**src/extension.ts(16,5): error TS2305: Module ''vscode'' has no exported member 'WebviewPanel'.
src/extension.ts(120,26): error TS2339: Property 'createWebviewPanel' does not exist on type 'typeof window'.
src/provider.ts(13,5): error TS2305: Module ''vscode'' has no exported member 'WebviewPanel'.**

If I bump the vscode egine version up to ^1.23.0 the prepublish step runs fine.  Did you run into a similar error?

Thanks

Reply:
You're right! It's required to up vscode engine version to 1.23.0